RecoLocalTracker/SiStripClusterizer/plugins/ClustersFromRawProducer.cc

```diff
@@ -355,9 +355,28 @@ try {
 	  }                                               
 	  continue;
 	}
-      } else {
+    } else if likely(mode == sistrip::READOUT_MODE_ZERO_SUPPRESSED_LITE8 ) {
+
+      try {
+        // create unpacker
+        sistrip::FEDZSChannelUnpacker unpacker = sistrip::FEDZSChannelUnpacker::zeroSuppressedLiteModeUnpacker(buffer->channel(fedCh));
+
+        // unpack
+        clusterizer.addFed(unpacker,ipair,record);
```

Comment:
you need to merge PR  #12307 
if you collapse Lite8 and lite10 there will be no problem..